protocol/contracts/reserve/src/state/config.rs

```diff
@@ -34,40 +33,32 @@ impl Config {
         Self::STORAGE.load(storage).map_err(Into::into)
     }
 
-    pub fn update_lease_code(storage: &mut dyn Storage, lease_code_id: CodeId) -> Result<()> {
+    pub fn update_lease_code(storage: &mut dyn Storage, lease_code: Code) -> Result<()> {
         Self::STORAGE
-            .update(storage, |_config: Self| Ok(Self { lease_code_id }))
+            .update(storage, |_config: Self| Ok(Self::new(lease_code)))
             .map(|_| ())
     }
 }
 
-impl From<InstantiateMsg> for Config {
-    fn from(msg: InstantiateMsg) -> Self {
-        Self {
-            lease_code_id: msg.lease_code_id.into(),
-        }
-    }
-}
-
 #[cfg(test)]
 mod test {
     use cosmwasm_std::{testing::MockStorage, Storage};
-    use platform::contract::CodeId;
+    use platform::contract::{Code, CodeId};
 
     use crate::Config;
 
     #[test]
     fn store_load() {
-        let lease_code_id = 12;
+        let lease_code = Code::unchecked(12);
```

Comment:
I'd like to reiterate on the magic numbers, as the others at least were `>= 1000`.

Reply:
I do not understand the problem here - it's just a test that should get some input data.

Reply:
My point is that, the black box integration tests /`./tests`/ for example uses the normal numbering because it goes through real API. In this case, we're adding magic numbers which if not hard to distinguish from other components, could be brushed off. That's the reason when I use magic numbers, I use `0xDEADC0DE`. It's enormous and there isn't a way to mistake it.

The other tests at least used larger integers.